.github/workflows/docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Markdown Linter
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+  pull_request:
+    paths-ignore:
+      - './**/*.md'
+      - './.vscode/**'
+      - './.husky/**'
+jobs:
+  test:
+    permissions:
```

Comment:
If we have a single `job`, let’s move `permission` to the root level as we have in another workflows with one job.